schemes/rrtmgp/rrtmgp_pre.meta

```diff
@@ -0,0 +1,301 @@
+[ccpp-table-properties]
+  name = rrtmgp_pre
+  type = scheme
+  dependencies = ./objects/ccpp_fluxes_byband.F90
+
+[ccpp-arg-table]
+  name  = rrtmgp_pre_init
+  type  = scheme
+[ nradgas ]
+  standard_name = number_of_active_gases_for_RRTMGP
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ available_gases ]
+  standard_name = gas_concentrations_rrtmgp_object_for_RRTMGP
+  units = none
+  type = ty_gas_concs_ccpp
+  dimensions = ()
+  intent = out
+[ gaslist ]
+  standard_name = list_of_active_gases_for_RRTMGP
+  units = none
+  type = character | kind = len=5
+  dimensions = (gaslist_dimension)
+  intent = in
+[ gaslist_lc ]
+  standard_name = lowercase_list_of_active_gases_for_RRTMGP
+  units = none
+  type = character | kind = len=5
+  dimensions = (number_of_active_gases_for_RRTMGP)
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = none
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = out
+
+[ccpp-arg-table]
+  name  = rrtmgp_pre_timestep_init
+  type  = scheme
+[ ncol ]
+  standard_name = horizontal_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ coszrs ]
+  standard_name = cosine_of_solar_zenith_angle_for_radiation
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = in
+[ nstep ]
+  standard_name = current_timestep_number
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ dtime ]
+  standard_name = timestep_for_physics
+  units = s
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ iradsw ]
+  standard_name = frequency_of_shortwave_radiation_calculation
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = in
+[ irad_always ]
+  standard_name = number_of_timesteps_to_force_radiation_calculation_after_initialization
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ offset ]
+  standard_name = number_of_seconds_until_next_shortwave_radiation_timestep
+  units = s
+  type = integer
+  dimensions = ()
+  intent = out
+[ idxday ]
+  standard_name = daytime_points
```

Comment:
I am wondering if we should replace `points` with `columns` here, as I don't think there is a good standard name definition of "points" in this context.  So basically:


```suggestion
  standard_name = daytime_columns
```

This same request holds for the other daytime and nighttime `points` variables below as well.